### PR TITLE
Added a new haiku in do_mba.md file

### PR DESCRIPTION
**Pull request by fcpimenta**
My MBA in Cloud Computing with DigitalOcean

### CODE REVIEW

File: _haikus/do_mba.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: do_mba
+author: fcpimenta
+---
+
+This year I made my <br>
+MBA with DigitalOcean cloud <br>
```
Comment (ltagliaferri) [inline _haikus/do_mba.md:8]:
```suggestion
MBA with DO cloud <br>
```

File: _haikus/do_mba.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+instead AWS and Azure <br>
```
Comment (ltagliaferri) [inline _haikus/do_mba.md:9]:
```suggestion
instead AWS <br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Hi Fernando, MBA and AWS would count as three syllables each (as in "M-B-A" and "A-W-S". I made suggestions to change it to conform to the syllable count :)

File: _haikus/do_mba.md (hunk 2 above)
Comment (fcpimenta) [reply _haikus/do_mba.md:9]: Hi Lisa, as always, As always, you shine and help us in any way you can. Thank you!!

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**